contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--5.0.0--5.1.0.sql

```diff
@@ -5486,6 +5486,95 @@ BEGIN
 END;
 $$;
 
+DO $$
+DECLARE
+    exception_message text;
+BEGIN
+    ALTER FUNCTION sys.babelfish_try_conv_float_to_string(TEXT, FLOAT, NUMERIC) RENAME TO babelfish_try_conv_float_to_string_deprecated_in_5_1_0;
```

Comment:
Why are we deprecating if all we are doing is replacing the body of the function? Are we changing any input argument type or return type?